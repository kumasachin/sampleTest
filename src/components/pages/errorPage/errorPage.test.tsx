import React from 'react';
import ErrorPage from './errorPage';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('renders correctly enzyme', () => {
    const wrapper = shallow(<ErrorPage />)

    expect(toJson(wrapper)).toMatchSnapshot();
});