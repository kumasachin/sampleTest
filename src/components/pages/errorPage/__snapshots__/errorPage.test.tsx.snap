// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Container))>
    Api Failure. Please retry by refreshing
  </WithStyles(ForwardRef(Container))>
</Fragment>
`;
